arangod/Network/Methods.cpp

```diff
@@ -351,6 +353,25 @@ class RequestsState final : public std::enable_shared_from_this<RequestsState> {
     }
   }
 
+  void errorResponse(arangodb::Result&& result) {
+    TRI_ASSERT(result.fail());
+    Scheduler* sch = SchedulerFeature::SCHEDULER;
+    if (_options.skipScheduler || sch == nullptr) {
+      _promise.setException(arangodb::basics::Exception(std::move(result), __FILE__, __LINE__));
```

Comment:
hm, will this be another time where we allways have the same file/line in errormessages? shouldn't they be put in as param to capture the place this error was generated?

Reply:
Good Spot, thank you for checking.
I will move the file, line, combitionation out of this helper function to the caller, however I think It should stay there and not be forwarded to another caller level (so it will still be this file, just a different line) as the communication error is caused here, not by the caller of it. :+1: